bsd/models.py

```diff
@@ -64,6 +65,11 @@ def find_constituents_by_state_cd(state_cd):
     filter = {}
     filter['state_cd'] = str(state_cd)
     filter['is_subscribed'] = True
+
+    """Check if member of sendable cons group"""
```

Comment:
comment is out of date. just check if cons group param is present and add to filter if not None, or something like that